### DISCUSSION TITLE
Helm chart as part of Glowroot project

### DISCUSSION THREAD

**Discussion by robinroos** [glowroot/glowroot#973] [General]
I use the Helm chart available here: https://novumrgi.github.io/helm/charts/glowroot/

Because of the way the chart needs to drive the Glowroot Central installation that chart works only with the novumrgi/glowroot-central image of Glowroot, and not with the official distribution.  This makes it tricky to on-board updates; I am reliant on someone else (Lucas Hahn) kindly to accommodate updates in the novumrgi repos.

Is there any progress regarding on-boarding that Helm chart into the Glowroot repo at github, and merging the PRs necessary for it to work with the standard distribution?

**Comment by Never-lab** [comment]
Thanks for raising this — there was still **no official Helm chart** in upstream when this thread started; Central was distributed as a JAR (+ wiki/Docker notes), and K8s packaging remained community-driven.

The [novumrgi chart](https://novumrgi.github.io/helm/charts/glowroot/) dependency on `novumrgi/glowroot-central` vs upstream releases is exactly the friction you describe.

**Update:** an upstream PR is now open to address this — **[#1224 — Official glowroot-central Helm chart](https://github.com/glowroot/glowroot/pull/1224)** (closes #729). It adds a minimal reference chart under `helm/glowroot-central` using the stock `glowroot/glowroot-central` image (no forked image), external Cassandra via values/Secret, UI + gRPC Services, optional Ingress.

Feedback welcome on the PR — especially from anyone running the novumrgi chart today.
